### ISSUE TITLE
Documentation fails building on docs.rs

### ISSUE THREAD

**Issue by hug-dev** [parallaxsecond/rust-tss-esapi#211]
Since the latest releases of `tss-esapi` and `tss-esapi-sys`: [see this](https://docs.rs/crate/tss-esapi/5.0.0/builds/365605)

```
[INFO] [stderr] error: failed to run custom build command for `tss-esapi-sys v0.1.0`
[INFO] [stderr] 
[INFO] [stderr] Caused by:
[INFO] [stderr]   process didn't exit successfully: `/opt/rustwide/target/debug/build/tss-esapi-sys-7a0937d3c8a6cdbf/build-script-build` (exit code: 101)
[INFO] [stderr]   --- stdout
[INFO] [stderr]   cargo:rerun-if-env-changed=TSS2_ESYS_NO_PKG_CONFIG
[INFO] [stderr]   cargo:rerun-if-env-changed=PKG_CONFIG
[INFO] [stderr]   cargo:rerun-if-env-changed=TSS2_ESYS_STATIC
[INFO] [stderr]   cargo:rerun-if-env-changed=TSS2_ESYS_DYNAMIC
[INFO] [stderr]   cargo:rerun-if-env-changed=PKG_CONFIG_ALL_STATIC
[INFO] [stderr]   cargo:rerun-if-env-changed=PKG_CONFIG_ALL_DYNAMIC
[INFO] [stderr]   cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-unknown-linux-gnu
[INFO] [stderr]   cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_unknown_linux_gnu
[INFO] [stderr]   cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
[INFO] [stderr]   cargo:rerun-if-env-changed=PKG_CONFIG_PATH
[INFO] [stderr]   cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-unknown-linux-gnu
[INFO] [stderr]   cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_unknown_linux_gnu
[INFO] [stderr]   cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
[INFO] [stderr]   cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
[INFO] [stderr]   cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-unknown-linux-gnu
[INFO] [stderr]   cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_unknown_linux_gnu
[INFO] [stderr]   cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
[INFO] [stderr]   cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
[INFO] [stderr] 
[INFO] [stderr]   --- stderr
[INFO] [stderr]   thread 'main' panicked at 'Failed to find tss2-esys library.: Failure { command: "\"pkg-config\" \"--libs\" \"--cflags\" \"tss2-esys\" \"tss2-esys >= 2.3.3\"", output: Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "Package tss2-esys was not found in the pkg-config search path.\nPerhaps you should add the directory containing `tss2-esys.pc\'\nto the PKG_CONFIG_PATH environment variable\nNo package \'tss2-esys\' found\nPackage tss2-esys was not found in the pkg-config search path.\nPerhaps you should add the directory containing `tss2-esys.pc\'\nto the PKG_CONFIG_PATH environment variable\nNo package \'tss2-esys\' found\n" } }', /opt/rustwide/cargo-home/registry/src/github.com-1ecc6299db9ec823/tss-esapi-sys-0.1.0/build.rs:30:14
[INFO] [stderr]   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[INFO] [stderr] warning: build failed, waiting for other jobs to finish...
[INFO] [stderr] error: build failed
```

**Comment by hug-dev** [comment]
We had this error in the beginning, a long time ago, I forgot how we solved it.

**Comment by ionut-arm** [comment]
We introduced a feature `docs` specifically for docs.rs that would disable some of the build steps so that linking with `tss-esapi`, for example, wouldn't happen (and we had those bindings commited, but we already had that)

**Comment by hug-dev** [comment]
Ah `parsec-service` docs also failed building because of the same problem 😬 When it is solved, it's worth to release a `5.0.1` and similar for Parsec.
